registration/include/pcl/registration/ndt.h

```diff
@@ -198,7 +198,7 @@ namespace pcl
       static void
       convertTransform (const Eigen::Matrix<double, 6, 1> &x, Eigen::Affine3f &trans)
       {
-        trans = Eigen::Translation<float, 3>(float (x (0)), float (x (1)), float (x (2))) *
+        trans = Eigen::Translation<float, 3>(x.head<3>().cast<float>()) *
```

Comment:
`convertTransform` here please

Reply:
It's the last definition of ```convertTransform```, and cannot be replaced.